### PR TITLE
Update run-a-validator.md

### PR DESCRIPTION
**Pull request by zktony**
## Describe your changes
Updated run-a-validator.md.

### CODE REVIEW

File: docs/run-a-validator.md [hunk 1]
```diff
@@ -24,6 +24,54 @@ For the full details of the standard hardware please see [here](https://github.c
...
+```
+Run a Parachain fullnode as a service
+```
+sudo systemctl daemon-reload
+sudo systemctl enable parachain
+sudo systemctl start parachain
+sudo systemctl status parachain
+```
+
+### Download Polkadex Solochain Binary
```
Comment (serhii-temchenko) [inline docs/run-a-validator.md:74]: Should this section contain instructions about solochain binary downloading?

**Review by serhii-temchenko — DISMISSED** [review]

File: docs/run-a-validator.md (hunk 1 above)
Comment (mt-gareth) [reply docs/run-a-validator.md:74]: I think it is pretty well covered in the next lines with:
`
curl -O -L https://github.com/Polkadex-Substrate/Polkadex/releases/download/v1.0.3/PolkadexNodeUbuntu.zip ; unzip PolkadexNodeUbuntu.zip`

I think you could clean up this line:
"Once you choose your cloud service provider and set-up your new server, the first thing you will do is install the necessary dependencies."

as this is no longer the first thing you are doing on a fresh install.

Comment (jaymansfield) [reply docs/run-a-validator.md:74]: Might want to point that to the latest rather then 1.0.3

File: docs/run-a-validator.md [hunk 2]
```diff
@@ -24,6 +24,54 @@ For the full details of the standard hardware please see [here](https://github.c
...
+```
+Download Polkadex Parachain Spec File
+```
+curl -O -L https://github.com/Polkadex-Substrate/parachain/releases/latest/download/polkadot-parachain-plain.json
+```
+Download Polkadot Spec File
+```
+curl -O -L https://github.com/Polkadex-Substrate/parachain/releases/latest/download/polkadot.json
+```
+Make Parachain bianry executable
```
Comment (Gauthamastro) [inline docs/run-a-validator.md]: typo - binary

Comment (zktony) [reply docs/run-a-validator.md]: done

File: docs/run-a-validator.md (hunk 1 above)
Comment (zktony) [reply docs/run-a-validator.md:74]: done

**Review by serhii-temchenko — APPROVED** [review]

**Comment by gabaod** [comment]
"You can then compare that to the current highest block via [Telemetry](https://telemetry.polkadot.io/#list/0x3920bcb4960a1eef5580cd5367ff3f430eef052774f78468852f7b9cb39f8a3c) or the [PolkadotJS Block Explorer](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fmainnet.polkadex.trade#/explorer)."     

Will we check our parachain node status at these exact same links? or is there new ones to add here.  thanks.

**Review by Gauthamastro — APPROVED** [review]

**Pull request merged**